ChronoStore/test/CMakeLists.txt

```diff
@@ -21,15 +18,19 @@ add_executable(hdf5_archiver_test
 )
 
 target_include_directories(hdf5_archiver_test PRIVATE
+    ${CMAKE_SOURCE_DIR}/ChronoStore/include
     ${CMAKE_SOURCE_DIR}/ChronoKeeper
     ${CMAKE_SOURCE_DIR}/chrono_common/include
     ${CMAKE_SOURCE_DIR}/Client/cpp/include
     ${CMAKE_SOURCE_DIR}/ChronoKeeper/include
+    ${HDF5_INCLUDE_DIRS}
     .
 )
 
-target_link_libraries(hdf5_archiver_test ${HDF5_LIBRARIES})
-target_link_libraries(hdf5_archiver_test thallium)
+target_link_libraries(hdf5_archiver_test 
+    ${HDF5_LIBRARIES} 
+    thallium
```

Comment:
other files link `thallium` first. I think it's better to be consistent with this ordering of linking. I am also okay to delay it to the next release. same thing applies to the ordering of include dirs.